content/blog/posts/mps-blog.md

```diff
@@ -0,0 +1,182 @@
+
+### Gems & Rounds
+
+{{< image-width src="/images/blog/mps-sample/image03.png" width="100%" alt="MPS Game Design.Gems & Rounds." >}} around the arena are floating gems of various colors. Picking these gems up, by walking through them, will award you points. The amount of points earned depends on the color of the gem. There are 5 gem types: Yellow (1 Point), Blue (3 Points), Green (7 points), Red (10 points), White (20 Points). These values are tunable from within the match controller component. Some of the more valuable gems can be found in hard-to-reach areas or areas where it is easy to fall out of the level, or both! We encourage you to explore the level and find all of the gems.
+
+Each match is comprised of multiple rounds that last 2 minutes each (defaults to 3). The number and duration of rounds is easily configurable in the match controller component. Gems respawn between rounds and each round can spawn different types of gems. By default, the per round gem spawn tables look like this:
+ 
+<table class="center"><style>
+table {
+    width: 400px;
+    border: 1px solid #CCC;
+}
+td {
+    border: 1px solid #CCC;
+}
+.small {
+    width: 1px;
+}
+.big {
+    width: 50%;
+}
+td > div {
+    white-space: nowrap;
+}
+.center {
+  margin-left: auto;
+  margin-right: auto;
+}
+</style>
+  <tr>
+    <th><center> Round 1 </th>
+    <th><center> Round 2 </th>
+    <th><center> Round 3 </th>
+  </tr>
+  <tr>
+    <td><center> Yellow</td>
+    <td><center> Blue</td>
+    <td><center> Green</td>
+  </tr>
+  <tr>
+    <td><center> Blue</td>
+    <td><center> Green</td>
+    <td><center> Red</td>
+  </tr>
+  <tr>
+    <td><center> Green</td>
+    <td><center> Red</td>
+    <td><center> White</td>
+  </tr>
+</table> 
+
+### PvE and Additional Content for the MPS
+
+The original concept of the MPS included traps that the player needed to avoid, as well as other gameplay enhancing items such as a second weapon, a jump pad and a teleporter platform. Ultimately some of these were not implemented for the initial release. We did, however, create all of the assets needed to add these items to the game. We’ve created GitHub Issues for each of these items and we encourage the O3DE community to make use of these assets to expand the MPS.
+
+* [MPS Expansion: Implement the Bubble Gun](https://github.com/o3de/o3de-multiplayersample/issues/228) 
+* [MPS Expansion: Implement the Jump Pad](https://github.com/o3de/o3de-multiplayersample/issues/227)
+* [MPS Expansion: Implement the Malfunctioning Shield Generator](https://github.com/o3de/o3de-multiplayersample/issues/226)
+* [MPS Expansion: Implement the Energy Collector](https://github.com/o3de/o3de-multiplayersample/issues/225) 
+* [MPS Expansion: Implement the Defense Turret](https://github.com/o3de/o3de-multiplayersample/issues/224)
+
+
+
+## Multiplayer Sample Configuration
```

Comment:
IMHO this is technical detail and should either:

* Be reduced in volume and link to the docs that provide more details
* Or be removed.

Reply:
This was modeled after another blog that had these types of implementation details. These instructions are also present here: https://github.com/o3de/o3de-multiplayersample/blob/stabilization/2305/Documentation/GamplayConfiguration.md so if you feel strongly that we should remove this section, we can.